client/signup/config/flows-pure.js

```diff
@@ -573,6 +573,15 @@ export function generateFlows( {
 			showRecaptcha: true,
 			hideProgressIndicator: true,
 		},
+		{
+			name: 'ecommerce-3y',
```

Comment:
I think this flow is not used anywhere?

Reply:
Probably not, but it appears to work and there's an `ecommerce-2y` so I decided to keep things consistent.